jobs/run-severity-generator.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#!/bin/bash
+
+# We use jupyter by default, but here we want to use python
+unset PYSPARK_DRIVER_PYTHON
+
+# Clone, install, and run
+git clone https://github.com/mozilla/python_mozetl.git
+cd python_mozetl
+pip install .
+python setup.py bdist_egg
+
+rm -rf output
+mkdir output
+
+spark-submit scheduling/tab_spinner.py
+
+# upload files to s3
+aws s3 sync ./output s3://telemetry-public-analysis-2/spinner-severity-generator/data/
```

Comment:
I don't have a good reason for this. It just happened this way as I moved the ipynb over.